DNN Platform/DotNetNuke.Abstractions/Logging/IEventLogConfigService.cs

```diff
@@ -0,0 +1,77 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information
+namespace DotNetNuke.Abstractions.Logging
+{
+    using System.Collections;
+    using System.Collections.Generic;
+
+    /// <summary>
+    /// The event log config service provides APIs for managing the
+    /// Event Log configuration and individual logs itself.
+    /// </summary>
+    public interface IEventLogConfigService
+    {
+        /// <summary>
+        /// Adds the log type.
+        /// </summary>
+        /// <param name="configFile">The configuration file.</param>
+        /// <param name="fallbackConfigFile">The fallback configuration file.</param>
+        void AddLogType(string configFile, string fallbackConfigFile);
+
+        /// <summary>
+        /// Adds the log type config info.
+        /// </summary>
+        /// <param name="logTypeConfig">The log type configuration.</param>
+        void AddLogTypeConfigInfo(ILogTypeConfigInfo logTypeConfig);
+
+        /// <summary>
+        /// Adds the log.
+        /// </summary>
+        /// <param name="logType">The log type.</param>
+        void AddLogType(ILogTypeInfo logType);
+
+        /// <summary>
+        /// Deletes the log type.
+        /// </summary>
+        /// <param name="logType">The log type.</param>
+        void DeleteLogType(ILogTypeInfo logType);
+
+        /// <summary>
+        /// Delete the log type config info.
+        /// </summary>
+        /// <param name="logTypeConfig">The log type config.</param>
+        void DeleteLogTypeConfigInfo(ILogTypeConfigInfo logTypeConfig);
+
+        /// <summary>
+        /// Gets the log type config info.
+        /// </summary>
+        /// <returns>The logs.</returns>
+        ArrayList GetLogTypeConfigInfo();
```

Comment:
I think it will make more sense to use an `IEnumerable<>` instead of `ArrayList`

```suggestion
        IEnumerable<ILogTypeConfigInfo> GetLogTypeConfigInfo();
```

Reply:
Definitely